bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -45,14 +45,26 @@ spec:
               --object-path /org/freedesktop/systemd1/unit/"$(svc_encode_name ${1})"_2eservice \
               --method org.freedesktop.systemd1.Unit.Restart "replace"
           }
+          # returns true if OVS is running in systemd
+          is_systemd_ovs(){
+            local state=$(gdbus call --system --dest org.freedesktop.systemd1 \
+              --object-path /org/freedesktop/systemd1/unit/openvswitch_2eservice \
+              --method org.freedesktop.DBus.Properties.Get \
+              org.freedesktop.systemd1.Unit ActiveState)
+            if [[ "$state" == *\'active\'* || "$state" == *\'activating\'* ]]; then
```

Comment:
shouldn't we match `reloading` as well, which is why we tried `ActiveEnterTimestamp > 0`